test/test_functionalization.py

```diff
@@ -744,6 +1044,21 @@ def forward(self, a_1):
     view_copy_default_1 = torch.ops.aten.view_copy.default(fill_scalar, [5, 5]);  fill_scalar = None
     add_tensor_1 = torch.ops.aten.add.Tensor(view_copy_default_1, 1)
     return (view_copy_default_1, add_tensor_1)
+    """)
+
+        reinplaced_logs = self.get_logs(f, torch.ones(8, 2), reapply_views=True, run_reinplace=True)
+        self.assertExpectedInline(reinplaced_logs, """\
+
+
+
+def forward(self, a_1):
+    add_tensor = torch.ops.aten.add.Tensor(a_1, 1);  a_1 = None
+    resize_default = torch.ops.aten.resize_.default(add_tensor, [5, 5])
+    view_default = torch.ops.aten.view.default(add_tensor, [25]);  add_tensor = None
+    fill_scalar = torch.ops.aten.fill_.Scalar(view_default, 1)
+    view_default_1 = torch.ops.aten.view.default(view_default, [5, 5]);  view_default = None
+    add_tensor_1 = torch.ops.aten.add.Tensor(view_default_1, 1)
+    return (view_default_1, add_tensor_1)
     """)
 
```

Comment:
I noticed I didn't see any tests in the set here for things like:

```
x = torch.zeros(10)
x[5] = 1
return x
```

Reply:
Good catch :D this breaks something - it looks like `FakeTensor` isn't handling the scalar-tensor conversion here well, when the `1` gets converted to a (non-fake) scalar-tensor.

Reply:
Oh I feel like this is an unrelated bug that @eellison is fixing right now, see https://github.com/pytorch/pytorch/pull/81927

Figure out some other way to do this for now. How about `x[5].fill_(1)`

Reply:
ah nice